src/baselaplace/predicting.jl

```diff
@@ -55,9 +61,12 @@ Computes predictions from Bayesian neural network.
 - `predict_proba::Bool=true`: If `true` (default), returns probabilities for classification tasks.
 
 # Returns
-
-- `fμ::AbstractArray`: Mean of the predictive distribution if link function is set to `:plugin`, otherwise the probit approximation. The output shape is column-major as in Flux.
-- `fvar::AbstractArray`: If regression, it also returns the variance of the predictive distribution. The output shape is column-major as in Flux.
+For classification tasks, LaplaceRedux provides different options:
+    -`normal_distr::Distributions.Normal`:the array of Normal distributions computed by glm_predictive_distribution  If the `link_approx` is set to :distribution
```

Comment:
@pat-alt i am confused on how to proceed here. the issue is that the output of the chain may have already passed through a softmax layer, if it's defined in the model.finaliser, so the output should not be converted again. should we add a check for this?

Reply:
Good point! I think we should probably add a check somewhere in the corestruct that contains the Flux chain. Let's open a separate issue for this.

Reply:
either that or modify the predict function so that it checks that the la.model has a finaliser layer